### ISSUE TITLE
Consolidate (Package Manager) Microsoft.AspNetCore.App references

### ISSUE THREAD

**Issue by WernerMairl** [loic-sharma/BaGet#84]
Bug

Currently the VS Package Manager UI shows that we have referenced multiple/different versions of 
"Microsoft.AspNetCore.App"
2.1.**0** and 2.1.**1**

AND a new Version (2.1.**2**) is global available but not used in our project.....

**Comment by loic-sharma** [comment]
Good catch! I'll upgrade BaGet to 2.1.2.

The 2.1.0 project, `BaGet.Tools.AzureSearchImporter`, will be folded into `BaGet` once the command line experience has been fleshed out. We won't need to worry about updating after that :)

**Comment by tomzo** [comment]
Once paket is merged in #103 this kind of multiple versions being references would be solved once for good.
